src/colorizer/ColorizeCanvas2D.ts

```diff
@@ -20,6 +16,9 @@ import {
   WebGLRenderer,
   WebGLRenderTarget,
 } from "three";
+import { Line2 } from "three/addons/lines/Line2.js";
```

Comment:
Replaced with the Line2 component, which can have variable width.

Note that this will probably be replaced again with the LineSegments2 class later in order to handle discontinuities.